modules/op-release-notes-1-10.adoc

```diff
@@ -211,4 +211,21 @@ With this update, {pipelines-title} General Availability (GA) 1.10.4 is availabl
 
 * Before this update, the `pipelinerun.timeouts` field was reset to the `timeouts.pipeline` value, ignoring the `timeouts.tasks` and `timeouts.finally` values. This update fixes the issue and sets the correct default timeout value for a `PipelineRun` resource.
 
-* Before this update, the controller logs contained unnecessary data. This update fixes the issue.
\ No newline at end of file
+* Before this update, the controller logs contained unnecessary data. This update fixes the issue.
+
+[id="release-notes-1-10-5_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.5
+
+With this update, {pipelines-title} General Availability (GA) 1.10.5 is available on {product-title} 4.10 in addition to 4.11, 4.12, and 4.13.
+
+[IMPORTANT]
+====
+{pipelines-title} 1.10.5 is only available in the `pipelines-1.10` channel on {product-title} 4.10, 4.11, 4.12, and 4.13. It is not available in the `latest` channel for any {product-title} version.
+====
+
+[id="fixed-issues-1-10-5_{context}"]
+=== Fixed issues
+
+* Before this update, huge pipeline runs were not getting listed or deleted using the `oc` and `tkn` commands. This update mitigates this issue by compressing the huge annotations that were causing this problem. Keep in mind that if the pipeline runs are still too huge after compression, then the same error still recurs.
```

Comment:
```suggestion
* Before this update, huge pipeline runs were not getting listed or deleted using the `oc` and `tkn` commands. This update mitigates this issue by compressing the huge annotations that were causing this problem. Remember that if the pipeline runs are still too huge after compression, then the same error still recurs.
```
Vale flags "keep in mind" but I couldn't find the place in the style guide that mentions it. I would recommend using "remember". 

Also, I am curious, is "huge" the correct terminology here? Or could it be large? Only reason is I did a quick search of the docs and noticed huge was only used in reference to page size.

Reply:
Hello @opayne1 thank you for the comments, I tried to replace the word "huge" for RNs before, but the engineer wanted me to use the word "huge" so that is why I used the terminology here.

For now, since the word is already used in our documentation ([here](https://docs.openshift.com/container-platform/4.13/cicd/pipelines/op-release-notes.html#release-notes-1-9-3_op-release-notes)) and we need to release this as soon as possible, I would prefer if we could merge it, if you are okay with it. But next week I will discuss it with the engineers and see if I can replace the word.

I integrated the rest of the changes so I hope it is ready for merge. Thank you!